### PR TITLE
K-9 navBar Component creation/implementation

### PR DESCRIPTION
**Pull request by sduval215**

### CODE REVIEW

File: client/src/components/search.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            placeholderText: 'Enter a member name..',
+            searchType: 'member',
+            inputValue: '',
+            searchResults: InputSearchService.fetchSearch('members')
+        }
+        this.setPlaceholder = this.setPlaceholder.bind(this);
+        this.updateInputValue = this.updateInputValue.bind(this);
+    }
+
+//FUNCTION TO BE FIRED ON SELECTOR CHANGE
```
Comment (greenac) [inline client/src/components/search.js]: Javascript has a block comment syntax that should be used for any method/function description. When you run something like jsdoc these will be pulled into the documentation automatically. It's the standard.

```
/**
* some comments
* some other comments
*/
```

File: client/src/components/search.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+//CHECKS TARGET VALUE AND SETS PLACEHOLDER AND SEARCH TYPE STATE
+    setPlaceholder(event){
+        var target = event.target.value;
```
Comment (greenac) [inline client/src/components/search.js]: I think you're already running es6, right? You're using a class definition for this component. If that is the case, you should not use `var` anymore. Use `let` or `const` for variable declaration.

File: client/src/components/search.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        if(target === 'members'){
```
Comment (greenac) [inline client/src/components/search.js]: most linters will throw an error with this format. The standard is like:
```
if (target === 'members') {
    // some logic
} else {
    // some other logic
}
```

File: client/src/components/search.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            this.setState({
+                placeholderText: 'Enter a member name..',
+                searchType: target,
+                searchResults: InputSearchService.fetchSearch(target)
+            })
+        }else{
```
Comment (greenac) [inline client/src/components/search.js]: see https://github.com/kapitol-app/kapitol-platform/pull/5/files#diff-86bed7783fd88b79b9c4f6a49f8b92b2R24

**Review by greenac — COMMENTED** [review]
I left a few comments @sduval215 if you can check them please.

File: client/src/components/search.js (hunk 3 above)
Comment (sduval215) [inline client/src/components/search.js]: Isn't that how it's written here?

**Pull request merged**